workspaces/meewee-time-registration/plugins/plugin-meewee-time-registration/meewee-time-registration/src/axios/index.ts

```diff
@@ -0,0 +1,70 @@
+// TESTO
+import axios, { AxiosError, AxiosRequestConfig, AxiosResponse } from 'axios';
```

Comment:
We want to use a consistent library here and there is an ADR to follow: https://backstage.io/docs/architecture-decisions/adrs-adr013